gateway/1.9/modules/ROOT/pages/flex-gateway-managed-getting-started.adoc

```diff
@@ -56,12 +55,12 @@ https://jsonplaceholder.typicode.com/
 +
 The UI takes you to the *API Summary* page of the newly created API instance.
 +
-Inside the private space the Managed Flex Gateway is deployed in, all HTTP requests made to `localhost` on port `8081` are now proxied to the `jsonplaceholder` service.
-. To test the API Instance, run the following command in a new terminal window:
+All HTTP requests made to the public endpoint are now proxied to the `jsonplaceholder` service.
+. To test the API instance, replace the public endpoint placeholder and run this command in a new terminal window:
 +
 [source,ssh]
 ----
-curl -s -o /dev/null -w "%{http_code}\n" --request GET 'http://localhost:8081/'
+curl -s -o /dev/null -w "%{http_code}\n" --request GET 'https://<publicEndpoint>'
```

Comment:
Given the URL that appears in the Public Endpoint block already contains the `https://` schema, maybe we should omit it.
For clarity, maybe we can specify a format of the domain, so it is clear that it should something similar to:
`https://my-gateway-abc123.edf456.us-east1.anypoint.mulesfot.com/`